app/controllers/follows_controller.rb

```diff
@@ -30,16 +30,25 @@ def create
                  else
                    User.find(params[:followable_id])
                  end
+
```

Comment:
Another method for malicious behavior here would be if someone passed in 500+ followable_ids. Might be a good idea to `raise RateLimitChecker::DailyFollowAccountLimitReached` above before we even do the followable lookup if `params[:followable_id].count > 500`

Reply:
That sounds like a good idea.